test/ParseableInterface/inherited-defaults-checks.swiftinterface

```diff
@@ -0,0 +1,38 @@
+// swift-interface-format-version: 1.0
+// swift-module-flags: -module-name InheritedDefaults
+
+// RUN: %empty-directory(%t)
+// RUN: %target-typecheck-verify-swift
```

Comment:
What's this extra step buying us?

Reply:
Oops – only meant to keep the first line.